dev-php/pecl-eio/metadata.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<maintainer type="person" proxied="yes">
-<email>rrosmanov@gmail.com</email>
-<name>Ruslan Osmanov</name>
-</maintainer>
-<maintainer type="project" proxied="proxy">
-<email>proxy-maint@gentoo.org</email>
-<name>Proxy Maintainers</name>
-</maintainer>
+  <maintainer type="person" proxied="yes">
+    <email>rrosmanov@gmail.com</email>
+    <name>Ruslan Osmanov</name>
+  </maintainer>
+  <maintainer type="person" proxied="yes">
+    <email>608192+rosmanov@users.noreply.github.com</email>
+    <name>Ruslan Osmanov</name>
```

Comment:
Any reason you're adding yourself twice here?

Also, please use non-noreply for git author/signed off by

Reply:
> Any reason you're adding yourself twice here?

I wanted to use my no-reply email.

Reply:
No-reply emails have been removed.